### PR TITLE
added info statement to methods in vdm

### PR DESCRIPTION
**Pull request by clintonskitson**
```
time="2015-09-20T05:06:05Z" level=info msg="docker: Mount()" newFsType= overwriteFs=false volumeID= volumeName=test2 
time="2015-09-20T05:06:06Z" level=info msg="Got next device name: /dev/xvdd" 
time="2015-09-20T05:06:06Z" level=info msg="Waiting for volume attachment to complete" 
time="2015-09-20T05:06:12Z" level=info msg="Attached volume vol-6484b59d to instance i-550be190" 
time="2015-09-20T05:06:19Z" level=info msg="docker: Unmount()" volumeID= volumeName=test2 
time="2015-09-20T05:06:20Z" level=info msg="Waiting for volume detachment to complete" 
time="2015-09-20T05:06:30Z" level=info msg="Detached volume vol-6484b59d" 
time="2015-09-20T05:08:37Z" level=info msg="docker: Create()" volumeName=testxx1 volumeOpts=map[volumeOpts:map[size:2]] 
time="2015-09-20T05:08:38Z" level=info msg="Waiting for volume creation to complete" 
time="2015-09-20T05:08:40Z" level=info msg="docker: Path()" volumeID= volumeName=testxx1 
```

### CODE REVIEW

File: volume/volume.go [hunk 1]
```diff
@@ -108,6 +120,9 @@ func (vdm *VolumeDriverManager) Path(volumeName, volumeID string) (string, error
 
 func (vdm *VolumeDriverManager) Create(volumeName string, volumeOpts VolumeOpts) error {
 	for _, driver := range vdm.Drivers {
+		log.WithFields(log.Fields{
+			"volumeName": volumeName,
+			"volumeOpts": golf.Fore("volumeOpts", volumeOpts)}).Debugf("%s: Create()", driver.Name())
```
Comment (akutz) [inline volume/volume.go]: Hi @clintonskitson,

This is not necessary. The `logrus` branch we use automatically unwinds fields using `Golf`. You don't need to reference `Golf` at all anywhere in this file, and the package import should be removed.

File: volume/volume.go [hunk 2]
```diff
@@ -3,6 +3,7 @@ package volume
 import (
 	log "github.com/Sirupsen/logrus"
 
+	"github.com/akutz/golf"
```
Comment (akutz) [inline volume/volume.go]: Hi @clintonskitson,

See this [note](https://github.com/emccode/rexray/pull/67/files#r39933268).

File: volume/volume.go [hunk 3]
```diff
@@ -87,6 +88,11 @@ func getDrivers(
 
 func (vdm *VolumeDriverManager) Mount(volumeName, volumeID string, overwriteFs bool, newFsType string) (string, error) {
 	for _, driver := range vdm.Drivers {
+		log.WithFields(log.Fields{
+			"volumeName":  volumeName,
+			"volumeID":    volumeID,
+			"overwriteFs": overwriteFs,
+			"newFsType":   newFsType}).Debugf("%s: Mount()", driver.Name())
```
Comment (akutz) [inline volume/volume.go]: Hi @clintonskitson,

I'm only going to comment on this once for this file, but it's applicable in *numerous* places. The `Debugf` and all the `LOGf` functions in `logrus` exist for drop-in compatibility with `stdlib`'s `log` package only. All places where the format variants are used should be changed. For example, the above line should look like this:

```
		log.WithFields(log.Fields{
			"volumeName":  volumeName,
			"volumeID":    volumeID,
			"overwriteFs": overwriteFs,
			"newFsType":   newFsType,
			"driverName": driver.Name()}).Debu("mounting volume")
```

However, if you'll look inside the providers, I took great pains to log most everything at the provider level as possible as all log statements always include the `providerName` field in conjunction with that provider's constant value denoting the same.

**Comment by clintonskitson** [comment]
@akutz, thanks for the eyes.  Updated based on comments.

**Pull request merged**